otter/convergence/service.py

```diff
@@ -180,13 +180,15 @@ def all_met(server, current_lb_nodes):
                              if node.matches(server)]))
 
 
-def update_servers_cache(group, now, servers, lb_nodes, include_deleted=True):
+def update_servers_cache(group, now, servers, lb_nodes, lbs,
```

Comment:
Eventually I figured out how `update_servers_cache` managed to change its signature without `update_servers_cache` showing up anywhere in the diff, so I don't actually have feedback about this change.  However, I will take this opportunity to say that this is kind of pointlessly confusing, and `ConvergenceExecutor` really wants to be an interface, and `launch_server_executor` and `launch_stack_executor` want to be providers of it.  This might be a place where `@provider` might be more appropriate than `@implementer` because they're guaranteed to be stateless, but nevertheless, idiomatic convention in Python is to group related functions in a namespace by making classes or modules, not making "bags" of functions that require said functions to be defined under misleadingly distinct names :)